implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpMeterRegistry.java

```diff
@@ -135,6 +135,15 @@ public void start(ThreadFactory threadFactory) {
             this.meterPollingService.scheduleAtFixedRate(this::pollMetersToRollover, getInitialDelay(),
                     config.step().toMillis(), TimeUnit.MILLISECONDS);
         }
+
+        if (config.enabled()) {
+            sayHello();
+        }
+    }
+
+    private void sayHello() {
+        logger.info("Publishing metrics for {} to {} with resource attributes {}", getClass().getSimpleName(),
```

Comment:
Right now this will result in two log messages on start. One from `PushMeterRegistry` like:
```
publishing metrics for OtlpMeterRegistry every 1m
```

And now this also.

```
Publishing metrics for OtlpMeterRegistry to http://localhost:50901/v1/metrics with resource attributes {}
```

I think it may be confusing to users to see both messages. They may think two registries are publishing. I'll open a proposal to make the log message on start from PushMeterRegistry customizable and then this can be rebased on top of that if we merge it.

Reply:
#4848 is now merged and these changes can be adapted to use that.